frontend/src/components/FileInput.css

```diff
@@ -0,0 +1,10 @@
+.fileIcon {
+  display: inline-block;
+  height: 0%;
+  vertical-align: middle;
+  padding-left: 30%;
+}
+
+#fileIconLabel {
+  margin-top: 0;
+}
```

Comment:
No need to create this class. You can use the Utility classes that USWDS already has: https://designsystem.digital.gov/utilities/margin-and-padding. 

In this case, you should be able to add `margin-top-0` to your label: 
```jsx
<label className="usa-label margin-top-0" htmlFor="fileIconDiv">
   {props.fileName}
</label>
```

Reply:
good call.